cli/hzn.go

```diff
@@ -374,6 +374,9 @@ Environment Variables:
 	userinputRemoveForce := userinputRemoveCmd.Flag("force", msgPrinter.Sprintf("Skip the 'Are you sure?' prompt.")).Short('f').Bool()
 
 	serviceCmd := app.Command("service", msgPrinter.Sprintf("List or manage the services that are currently registered on this Horizon edge node."))
+	serviceLogCmd := serviceCmd.Command("log", msgPrinter.Sprintf("Show the container logs for a service."))
+	logServiceName := serviceLogCmd.Arg("service", msgPrinter.Sprintf("Service name. Use the url field of a service definition. Similar to tail behavior")).String()
```

Comment:
This is not the command structure I wanted. I think we want the service name to be required and the -f to be an optional flag that enables log rotation and continuous output.
logServiceName := serviceLogCmd.Arg("service", msgPrinter.Sprintf("The name of the service whose log records should be displayed. The service name is the same as the url field of a service definition. Displays log records similar to tail behavior and returns .")).Required().String()
logServiceNameTail := serviceLogCmd.Flag("service", msgPrinter.Sprintf("Continuously polls the service's logs to display the most recent records, similar to tail -F behavior.")).Short('f').Bool()